nsfc-plugin/src/test/java/org/osc/controller/nsfc/OSGiIntegrationTest.java

```diff
@@ -265,7 +267,10 @@ public void setup() {
 
         this.osClient = v3.authenticate();
```

Comment:
i am confused. does the osgi integration test require a working openstack installation? This is not the case for other OSGI integration tests. i would setup a dummy server via java code.

see https://github.com/arvindn05/security-mgr-nsm-plugin/blob/master/src/test/java/com/mcafee/nsm/AbstractNsmTest.java